README.md

```diff
@@ -115,8 +115,11 @@ Permissions that applications may use (names are subject to change):
 | Music | Access to Music directory, playlists and coverart cache. |
 | Phone | Make Phone calls, either directly or through system voice call UI. |
 | Pictures | Access to Pictures directory and thumbnails. |
+| PublicDir | Access to Public directory. |
 | Sharing | Use of sharing pages in the application to share data via Bluetooth, email or other accounts. |
 | Synchronization | Access to synchronization framework. |
+| TemplatesDir | Access to Templates directory. |
+| UserDirs | Access to XDG user directories: Documents, Downloads, Music, Pictures, Public, and Templates (except Desktop that is unused in Sailfish OS). |
```

Comment:
nit: Feel free to ignore. That "(except ..)" portion feels a bit incomplete or something. Maybe  "all except..." or "Desktop is not include because..." or something.

Reply:
Yeah I was thinking if it is reasonable to mention as it is excluded in the list already. Maybe for clarity I just drop that ()-content.

Reply:
Dropped the excess content with: 864cd19 - I guess it is better now.